packages/cache/src/internal/downloadUtils.ts

```diff
@@ -247,7 +249,12 @@ export async function downloadCacheStorageSDK(
 
     try {
       downloadProgress.startDisplayTimer()
-
+      const abortTimeInMs =
```

Comment:
Better to handle undefined `abortTimeInMs` when instantiating `options`. This way all defaulting is in one place.

Reply:
We are doing the instantiation [here](https://github.com/actions/toolkit/blob/segment-download-timeout/packages/cache/src/options.ts#L95) while getting the options, but since the member is [declared optional](https://github.com/actions/toolkit/blob/segment-download-timeout/packages/cache/src/options.ts#L55) the undefined check is required.
<img width="1115" alt="Screenshot 2022-08-08 at 10 22 37 AM" src="https://user-images.githubusercontent.com/98868223/183341862-31455175-c1be-4808-81d3-5be8cd17ccc1.png">